invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/report_email.html

```diff
@@ -0,0 +1,28 @@
+{#
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# Invenio App RDM is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+-#}
+
+{% set BASE_URL = config.SITE_UI_URL %}
+
+{%- for entry in entries -%}
```

Comment:
could you attach a screenshot on how it renders as an email ?

Reply:
It can be found in the issue:

https://github.com/inveniosoftware/invenio-app-rdm/issues/1951#issuecomment-1326554003

Reply:
I am not sure how do I read the report then. Is all good with files? Do the admins who receive the report need to take any action ? The text is a bit vague, and I think it could be confusing - having in mind we are implementing it now as a part of the framework, we should target for clear messages

Reply:
The report only contains unhealthy files. 

About taking any action, perhaps @slint can give his insight on that point. In my (short) experience I never took action on files that were unhealthy.

Reply:
I agree the report is not very user-friendly (even for admins :) ).

The usual follow-up is to inspect the files in the shell and underlying filesystem and troubleshoot from there (most of the time it's false-positives, e.g. a check that timed-out). I'm not sure how much "insight" we can include in the report though since the only "extra" information we have comes from the `FileInstance.last_check` value which can be:

- `None`: means that the file check errored during the operation
- `False`: means that the produced checksum didn't match the originally stored checksum (that's pretty bad)

Reply:
I updated the template to contain:

- Last check date: `last_check_at`
- Lasts check result: `last_check`

I added a small text to help the reader (updated the screenshot).

Let me know your opinion about it :)